options/ansi/include/limits.h

```diff
@@ -3,6 +3,10 @@
 
 #define CHAR_BIT 8
 
+#ifndef MB_LEN_MAX
+# define MB_LEN_MAX 1
```

Comment:
Shouldn't this be 4 for UTF-8? I am not sure if we actually implement any of the conversions functions properly yet but when we do this should likely be 4.

Reply:
On Linux/glibc it appears to be 16... not sure.

Reply:
On musl it is 4, maybe it being bigger on glibc has something to do with locales.